docroot/modules/custom/foia_webform/src/FoiaSubmissionQueueingService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\foia_request\Entity\FoiaRequestInterface;
+
+/**
+ * Class FoiaSubmissionQueueingService.
+ */
+class FoiaSubmissionQueueingService implements FoiaSubmissionQueueingServiceInterface {
```

Comment:
Since the method below queues a `foiaRequest`, maybe we can call this `FoiaRequestQueuer` or `FoiaRequestQueuingService`